plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -356,7 +356,7 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
           }
         }
       val features = annotations.map {
-        val annotation = Feature.fromGeometry(it.geometry, it.jsonObject)
+        val annotation = Feature.fromGeometry(it.geometry, it.getJsonObjectCopy())
```

Comment:
If we make `jsonObject` internal below - can we use the object directly here? or is it also a potential bug?

Reply:
No, the purpose of adding a new method `getJsonObjectCopy` here is to avoid `ConcurrentModificationException`. While we updating the source, users can add or remove new properties.